<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalWindow</class>
 <widget class="QDialog" name="FinalWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>390</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutBotones" stretch="5,0,0,0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QComboBox" name="barraSeleccion">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>72</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>24</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>Seleccione Acci√≥n...</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Comprimir</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Eliminar</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mover</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnAplicar">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Aplicar</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnVolver">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Volver</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnTerminar">
      <property name="maximumSize">
       <size>
        <width>70</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string>Terminar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="layoutCabcera"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
